src/Sentry.AspNetCore/SentryMiddleware.cs

```diff
@@ -127,6 +127,9 @@ public async Task InvokeAsync(HttpContext context)
 
                 void CaptureException(Exception e)
                 {
+                    e.Data[Mechanism.HandledKey] = false;
+                    e.Data[Mechanism.MechanismKey] = "SentryMiddleware";
```

Comment:
Can we please differentiate between errors coming from unhandled exceptions and errors coming from `IExceptionHandlerFeature` ?

![image](https://user-images.githubusercontent.com/1633368/125165410-c6030d00-e164-11eb-92de-9ea96660c9bd.png)

Reply:
Maybe `Middleware.UnhandledException` (does it fit in the UI?)
And `IExceptionHandlerFeature`

Reply:
why not SentryMiddleware.UnhandledException ?

Reply:
it fits the UI (simulated crash)
![image](https://user-images.githubusercontent.com/8229322/125304644-23819000-e304-11eb-8876-5f78573d5a48.png)

Reply:
Set the Mechanism to "IExceptionHandlerFeature" and "SentryMiddleware.UnhandledException".
If the names are ok we are good to go :D